docs/markdown/Running-Meson.md

```diff
@@ -92,6 +92,21 @@ You can then open the generated solution with Visual Studio and
 compile it in the usual way. A list of backends can be obtained with
 `meson --help`.
 
+Environment Variables
+--
+
+Sometimes you want to add extra compiler flags, this can be done by
+passing them in environment variables when calling meson. See [the
+reference
+tables](Reference-tables.md#compiler-and-linker-flag-envrionment-variables)
+for a list of all the environment variables. Be aware however these
+environment variables are only used for native builds and will not
+affect the build when cross-compiling.
```

Comment:
They do affect the build, but only for selecting the native compiler. The cross compiler always comes from the cross file.

Reply:
Not only for selecting the native compiler, but also CFLAGS and friends are passed to native compiler too, AFAIK.

Reply:
Unrelated: just realized 'native' keyword argument is not documented in reference manual for executable().

Reply:
Is the current wording acceptable?